shell/common/platform_util_win.cc

```diff
@@ -392,6 +393,25 @@ bool MoveItemToTrash(const base::FilePath& path, bool delete_on_fail) {
          SUCCEEDED(pfo->PerformOperations());
 }
 
+bool GetFolderPath(int key, base::FilePath* result) {
+  wchar_t system_buffer[MAX_PATH];
+  system_buffer[0] = 0;
```

Comment:
Shouldn't it be okay to have the buffer uninitialized ?

Reply:
You'd think, but that's how Chromium does it. Happy to leave it uninitialized though!

Reply:
No problem in having it but if the call the failed it gonna point to some empty string and we don't use the result in the case, so its not necessary to initialize it here.